server/src/cloud/column.ts

```diff
@@ -65,6 +68,10 @@ export const initializeColumnFunctions = () => {
                 columns[request.columnId].name = request.name;
             }
 
+            if (request.color) {
```

Comment:
Maybe it would be reasonable to check if the given color is an actual color of the Color type in src/constants/colors.ts